### ISSUE TITLE
Assets\Plugins\SuperScience\Runtime\GizmoModule\Sample\SampleGizmos.cs(2,19): error CS0234: The type or namespace name 'SpatialTracking' does not exist in the namespace 'UnityEngine'

### ISSUE THREAD

**Issue by JustinSchneider** [LinceWorks/SimpleVR#1]
After cloning the project, installing dependencies and recommended plugins, adding and updating packages, I'm left with one error I can't resolve.

The SampleGizmos.cs file from SuperScience is unable to find the UnityEngine.SpatialTracking library from the package manager. I'm confident I've added the package to the SuperScience assembly as instructed.

Any idea or direction on what I'm missing here?

**Comment by dacaes** [comment]
Mmm it works for me. Do you have more info?

**Comment by JustinSchneider** [comment]
I managed to get it to go away after closing and re-opening the project a couple times. Not really clear on what the issue turned out to be. Might have just been assemblies being loaded incorrectly.

I did have to comment out the entirety of the DOTweenModuleUI.cs file on account of not being able to resolve the errors generated within, but I got the app to build and things seem to be functioning.
